### PR TITLE
api: Merge romulus terms api code into this branch so we have one source of truth

### PR DESCRIPTION
**Pull request by mattyw**

### CODE REVIEW

**Comment by cmars** [comment]
LGTM

File: api/wireformat/entities.go [hunk 1]
```diff
@@ -138,3 +138,51 @@ type CheckResult struct {
...
+	Revision  int       `json:"revision" yaml:"revision"`
+	CreatedOn time.Time `json:"created-on" yaml:"createdon"`
+	Content   string    `json:"content" yaml:"content"`
+}
+
+// CheckAgreementsRequest holds a slice of terms and the /v1/agreement
+// endpoint will check if the user has agreed to the specified terms
+// and return a slice of terms the user has not agreed to yet.
+type CheckAgreementsRequest struct {
+	Terms []string
```
Comment (alesstimec) [inline api/wireformat/entities.go]: `json:"terms"` just to be explicit

**Comment by alesstimec** [comment]
LGTM

**Comment by mattyw** [comment]
$$merge$$

**Pull request merged**